manifests/n/nzbget/nzbget/24.4/nzbget.nzbget.locale.en-US.yaml

```
@@ -0,0 +1,37 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: nzbget.nzbget
+PackageVersion: '24.4'
+PackageLocale: en-US
+Publisher: nzbget.net

```

```suggestion
Publisher: nzbget.com
```
Looks like I pressed the approve button too early. This value should exactly match what is written to Control Panel / Apps&Features / Registry post installation. I'll open a new PR to update this manifest value